egnoramoose.py

```diff
@@ -1,119 +1,167 @@
 class Pawn:
-	def __init__(self,location):
-		self.location = location
+    pass
+
+
+class Location:
+    def __init__(self, *location):
```

Comment:
This combined with :function `_convert` is where the magic happens. *param is the syntax for variable parameters. When you pass over multiple parameters, location becomes a tuple with parameter1 being the first parameter, and parameter2 being the second